src/vs/base/common/fuzzyScorer.ts

```diff
@@ -871,18 +876,19 @@ export function prepareQuery(original: string): IPreparedQuery {
 				}
 
 				values.push({
-					original: originalPiece,
+					original: originalPiece.replace('*', '').replace('?', ''),
 					originalLowercase: originalPiece.toLowerCase(),
 					pathNormalized: pathNormalizedPiece,
 					normalized: normalizedPiece,
 					normalizedLowercase: normalizedLowercasePiece,
-					expectContiguousMatch: expectExactMatchPiece
+					expectContiguousMatch: expectExactMatchPiece,
+					raw: originalPiece
 				});
 			}
 		}
 	}
-
-	return { original, originalLowercase, pathNormalized, normalized, normalizedLowercase, values, containsPathSeparator, expectContiguousMatch: expectExactMatch };
+	original = original.replace('*', '').replace('?', '');
```

Comment:
```suggestion
	original = original?.replace('*', '')?.replace('?', '');
```

Reply:
Thanks for taking a look at the code. But I don't see a need for using optional chaining here since original variable cannot be null nor undefined right ? I making the assumption based on the rest of the statements which use the same variable. Specially looking at line 851. Do you see otherwise ?